storybook/src/components/Button.js

```diff
@@ -0,0 +1,43 @@
+import React, {PureComponent} from 'react';
+import {CSSTransition} from 'react-transition-group';
+import "../css/Button.css";
+import classNames from 'classnames';
+
+export class Button extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            label: props.label,
+            round: props.round,
+            classNames: "custom-button ",
+            id: this.props.id,
+            disabled: this.props.disabled,
+        };
+        if(props.className) {
+            this.state.classNames = this.state.classNames + " " + props.className;
+        }
+
+        if(this.state.round) {
```

Comment:
You can use += here, but I think you should try to take advantage of the classNames module you imported above, I believe it has good support for this